src/main/kotlin/se/uulm/snowballr/backend/repository/association/AuthorOfPaperTableRepo.kt

```diff
@@ -0,0 +1,58 @@
+package se.uulm.snowballr.backend.repository.association
+
+import org.jetbrains.exposed.sql.SqlExpressionBuilder.eq
+import org.jetbrains.exposed.sql.selectAll
+import se.uulm.snowballr.backend.db.IDatabase
+import se.uulm.snowballr.backend.model.EntityType
+import se.uulm.snowballr.backend.model.SnowballRException.NotFoundException
+import se.uulm.snowballr.backend.model.dto.Author
+import se.uulm.snowballr.backend.model.dto.Paper
+import se.uulm.snowballr.backend.table.AuthorTable
+import se.uulm.snowballr.backend.table.PaperTable
+import se.uulm.snowballr.backend.table.association.AuthorOfPaperTable
+import se.uulm.snowballr.backend.table.toAuthor
+import java.util.UUID
+
+/**
+ * Defines an interface for repository operations related to the [AuthorOfPaperTable].
+ *
+ * This interface provides abstraction for handling persistence and retrieval
+ * operations for authors of papers. By using this interface, the functionality for
+ * managing author-paper associations can remain decoupled from the specifics of the database layer.
+ */
+interface IAuthorOfPaperTableRepo {
+    /**
+     * Returns all authors of a [Paper] or throws a [NotFoundException] if the paper with the passed [paperId] doesn't exist.
```

Comment:
This exceeds the maximum line length of 120. (we have no linter for comment line lengths)
```suggestion
     * Returns all authors of a [Paper] or throws a [NotFoundException] if the paper with the passed [paperId] doesn't
     * exist.
```